plugins/GSdx/res/tfx.fx

```diff
@@ -361,6 +370,11 @@ float4x4 sample_4p(float4 u)
 	return c;
 }
 
+float4 fetch_raw_color(int2 xy)
+{
+	int3: RawTexture.Load(int3(xy, 0));
```

Comment:
Just `RawTexture.Load(int3(xy, 0));` will do. ;)

Reply:
That did it! Finally no more crashes.

Now for the rest of the code and get the shader to be read properly :stuck_out_tongue: